besu/src/main/java/org/hyperledger/besu/services/BesuConfigurationImpl.java

```diff
@@ -21,13 +21,24 @@
 public class BesuConfigurationImpl implements BesuConfiguration {
 
   private final Path storagePath;
+  private final Path dataPath;
 
   public BesuConfigurationImpl(final Path storagePath) {
+    this(storagePath, storagePath);
+  }
+
+  public BesuConfigurationImpl(final Path storagePath, final Path dataPath) {
     this.storagePath = storagePath;
```

Comment:
Nit: since storage path is a subdirectory of datapath typically I would expect data path first in the args list.  However because of call evolution I understand if we want to leave it as second.

Reply:
clearly i hesitated a lot. both options are viable. i decided to put it last because of call evolution indeed.

Reply:
i don't have strong opinion on this. if you prefer i can switch args

Reply:
let's keep it as is for the moment